pkg/ocm/products.go

```diff
@@ -0,0 +1,20 @@
+package ocm
+
+import cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+
+const RosaProductId = "rosa"
+
+func (c *Client) GetTechnologyPreview(id string) (*cmv1.ProductTechnologyPreview, bool, error) {
+	response, err := c.ocm.ClustersMgmt().V1().
+		Products().Product(RosaProductId).
+		TechnologyPreviews().TechnologyPreview(id).
+		Get().
+		Send()
+	if response.Status() == 404 {
+		return nil, false, nil
+	}
+	if err != nil {
+		return nil, false, handleErr(response.Error(), err)
```

Comment:
this doesn't return the error to the user, I think we should still check in the caller since it may be something else